easybuild/easyconfigs/c/ctffind/ctffind-4.1.10-fosscuda-2018b.eb

```diff
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+#
+# Author: Ake Sandgren, HPC2N, Umea University
+
+easyblock = 'ConfigureMake'
+
+name = 'ctffind'
+version = '4.1.10'
+
+homepage = 'http://grigoriefflab.janelia.org/ctffind4'
+description = """Program for finding CTFs of electron micrographs."""
+
+toolchain = {'name': 'fosscuda', 'version': '2018b'}
+toolchainopts = {'openmp': True}
+
+source_urls = ['http://grigoriefflab.janelia.org/sites/default/files/']
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [
+    ('GSL', '2.5'),
+    ('LibTIFF', '4.0.9'),
+    ('wxWidgets', '3.0.4'),
+]
+
+separate_build_dir = True
```

Comment:
This can be dropped. See https://github.com/easybuilders/easybuild-easyconfigs/pull/8144#discussion_r283727812